<?xml version="1.0"?>


<!--EXAMPLE

roslaunch armadillo2 armadillo2.launch kinect:=true gazebo:=true moveit:=true world_name:="`rospack find armadillo2_gazebo`/worlds/objects_on_table.world" x:=0.3 y:=-0.35

roslaunch object_detection pick_and_place.launch  

rosservice call /pick_go "{}"
-->

<launch>

	<arg name="start_position" default="cobra_center" /> 
	<arg name="end_position" default="to_cheak" />
	<arg name="give_position" default="give" />
    

	<node name="find_object_node" pkg="obj_detector" type="alvar_markers_pub.py" output="screen">
	<param name="~items" value="['cup', 'bottle']" />
</node>

	<node name="pick" pkg="robotican_demos_upgrade" type="pick_can_with_feedback" output="screen" 				required="true">
		<param name="~start_position_name" value="$(arg start_position)"/>
		<param name="~end_position_name" value="$(arg end_position)"/>
		<param name="~object_name" value="can"/>
		<param name="~using_yolo" value="true" />
	</node>

    <node name="objects_handler" pkg="object_detection" type="objects_handler" output="screen">
        <param name="~db_path" value=""/>
    </node> 

	<node name="object_view" pkg="rqt_image_view" type="rqt_image_view" >
		<remap from="image" to="/yolo4_result/obj_pick/compressed" />
	</node>

</launch>


